lib/l10n_utils/__init__.py

```diff
@@ -175,3 +176,7 @@ def get_context_data(self, **kwargs):
     def render_to_response(self, context, **response_kwargs):
         return render(self.request, self.get_template_names(),
                       context, **response_kwargs)
+
+
+class L10nTemplateView(LangFilesMixin, TemplateView):
```

Comment:
I added this as a convenience as it should really already have existed and I wanted the docs to be easier to understand as I was reworking them.